media-video/subliminal/subliminal-2.0.3.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4} )
```

Comment:
[setup.py](https://github.com/Diaoul/subliminal/blob/master/setup.py#L65) claims support for python3_5. should we add it to the list?